app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="container d-flex justify-content-center align-items-center h-100">
+  <div class="col-md-6 col-lg-5">
+    <div class="card p-4 shadow-sm">
+      <h2 class="mb-3 text-center text-main font-semibold">Log in</h2>
+      <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+        <div class="mb-3">
+          <%= f.email_field :email, id: "email", class: "form-control", placeholder: "Email", autofocus: true, autocomplete: "email" %>
+        </div>
+        <div class="mb-3">
+          <%= f.password_field :password, id: "password", class: "form-control", placeholder: "Password" %>
+        </div>
+        <% if devise_mapping.rememberable? %>
+          <div class="form-group form-check">
+            <%= f.check_box :remember_me, class: "form-check-input", id: "remember-me" %>
+            <%= f.label :remember_me, class: "form-check-label", for: "remember-me" %>
+          </div>
+        <% end %>
+        <%= f.button "Log in", type: "submit", class: "btn btn-primary w-100 mt-3" %>
+      <% end %>
+      <div class="mt-3 text-center">
+        <%= render "devise/shared/links" %>
```

Comment:
### The same for login page 
- [ OPTIONAL ] Great job so far but I think you missed to style your sign in page the same as the given design so I highly recommend to style it the same since it is very important to have the same design as the given design but I am assign it as optional because the previous code reviewer doesn't mention it 😊 but kindly fix the styles here ✨✅♻🙏🏻🙏🏻

## There are alot of different things the nav contain the button for sign in  and arrow  in the original design✅

- Your design
![image](https://user-images.githubusercontent.com/58553711/221363293-b335e887-f61e-4fad-b0b0-f751f4a733d6.png)

- Original design 
![image](https://user-images.githubusercontent.com/58553711/221363444-c35d09a8-31e4-4fc4-ad46-83fdfea65b05.png)

Reply:
Hi @reem_lab, same goes for this page I will work on it.